generators/server/templates/_pom.xml

```diff
@@ -504,6 +512,11 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-test</artifactId>
```

Comment:
If it's redundant, we don't need it.
On Thu, Aug 4, 2016 at 14:38 Navid Ghahremani <notifications@github.com>
wrote:

> In generators/server/templates/_pom.xml
> <https://github.com/jhipster/generator-jhipster/pull/3900#discussion_r73598081>
> :
>
> > @@ -504,6 +512,11 @@
> >          </dependency>
> >          <dependency>
> >              <groupId>org.springframework.boot</groupId>
> > +            <artifactId>spring-boot-test</artifactId>
>
> Hi, I have one question, Why did you add this dependency? It is already
> inside of spring-boot-starter-test, is there any reason to do that?
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3900/files/35aa81bd29da21d6f709a009ed97266541e48dd2#r73598081>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABF5HdZt13PKAhxtpoagzKJoEameFePks5qck2xgaJpZM4JX4ec>
> .
>